main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    race_inst = None
+    guild_inst = None
+
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player in players.items():
+        if not Race.objects.filter(name=player["race"]["name"]).exists():
+            race_inst = Race.objects.create(
+                name=player["race"]["name"],
+                description=player["race"]["description"]
+            )
+
+        for skill in player["race"]["skills"]:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race_inst
+                )
+
+        guild = player.get("guild")
+        if guild is not None:
+            if not Guild.objects.filter(name=player["guild"]["name"]).exists():
+                guild_inst = Guild.objects.create(
+                    name=player["guild"]["name"],
+                    description=(
+                        player["guild"]["description"]
+                        if player["guild"]["description"] else None
+                    )
+                )
+        else:
+            guild_inst = None
```

Comment:
add a variant where the Race, Skill, Guild already exist in db so u just take the instance from db
it is really easy with method get_or_create - read about it